test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/actions/AuthActionSpec.scala

```diff
@@ -208,5 +219,24 @@ class AuthActionSpec extends ControllerSpec with MetricsMocks {
       ) mustBe Results.Ok
     }
 
+    "agents accessing an amendment screen without a selected client should be redirected back to plastics account" in {
+      // The plastics accounts UI will take the required actions; we will not duplicate them here
+      val agent = PptTestData.newAgent("456")
+      authorizedUser(
+        agent,
+        expectedPredicate = Some(
+          Enrolment(PptEnrolment.Identifier).and(CredentialStrength(CredentialStrength.strong))
+        )
+      )
+
+      val result =
+        amendmentAuthAction(new AllowedUsers(Seq.empty)).invokeBlock(authRequest(Headers(), agent),
+                                                                     okResponseGenerator
+        )
+
+      status(result) mustBe SEE_OTHER
+      redirectLocation(result) mustBe Some("/ppt-accounts-url")
```

Comment:
I'm struggling to see where that URL is coming from.

Reply:
Fake app config. Good point though; it's a side effect of a missing reset.